PhysicsTools/PatAlgos/python/slimming/miniAOD_tools.py

```diff
@@ -626,7 +626,17 @@ def miniAOD_customizeData(process):
     process.load("RecoPPS.Local.ctppsLocalTrackLiteProducer_cff")
     process.load("RecoPPS.ProtonReconstruction.ctppsProtons_cff")
     process.load("Geometry.VeryForwardGeometry.geometryRPFromDB_cfi")
+    process.load('L1Trigger.L1TGlobal.simGtExtFakeProd_cfi')
+    process.simGtExtFakeProd.bxFirst = cms.int32(-2)
+    process.simGtExtFakeProd.bxLast = cms.int32(2)
+    process.simGtExtFakeProd.setBptxAND = cms.bool(True)
+    process.simGtExtFakeProd.setBptxPlus = cms.bool(True)
+    process.simGtExtFakeProd.setBptxMinus = cms.bool(True)
+    process.simGtExtFakeProd.setBptxOR = cms.bool(True)
+    process.simGtExtFakeProd.tcdsRecordLabel = cms.InputTag("tcdsDigis","tcdsRecord")
```

Comment:
it's usually a very bad practice to import an object from somewhere and then edit it
(even worse yet to do it across the subsystems).
Can these settings be applied already in simGtExtFakeProd_cfi?
Alternatively, for miniAOD, there should be a clone with appropriately updated settings

Reply:
I agree.  So now your previous comment 

> * please drop type specifications for all parameters which already exist. This is a safer way to protect from parameter name mistakes and will also help in possible parameter migrations.

I should have understood as "drop type specifications for all parameters ... " after the object has been cloned.  That is what you meant ?

Reply:
Also, to understand better your comment, are you suggesting to have all the " "drop type specifications for all parameters ... " listed in simGtExtFakeProd_cfi, and not have them listed here at all, but instead here only call the load of the cloned module?

Reply:
in my earlier comment I somehow missed the point that this imports an existing module and modifies it; after you added more lines it became much more obvious that the modifications are in place.

as for the question to where and how to drop the type specifications, this refers to expressions that specify parameter values in all cases where the parameter is already known; these would be direct mods (like `a.par = value`), during cloning (like `b = a.clone(par = value)`), or in a modifier call (like `mod.toModify(a, par = value)`).